Makefile

```diff
@@ -1,5 +1,5 @@
 build:
-	GOOS=linux go build -ldflags="-s -w" -o bin/castai-agent .
+	GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build -ldflags "-s -w" -o bin/castai-agent-amd64 .
```

Comment:
does `docker build` work after renaming binary?

Reply:
It was broken. I only made it work for my machine. It's in a better state than it was before but obviously not ideal.